File: Python/1365_how_many_numbers_are_smaller_than_the_current_number.py
class Solution:
    def smallerNumbersThanCurrent(self, nums: list[int]) -> list[int]:
        count = 0
        ret_list = []
        for i in range(len(nums)):
            for j in range(len(nums)):
                if nums[i] > nums[j]:
                    count += 1
            ret_list.append(count)
            count = 0
        return ret_list
        

File: JavaScript/1365_how_many_numbers_are_smaller_than_the_current_number.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var smallerNumbersThanCurrent = function(nums) {
    let answer = [];
    let smallerNumbersCount = 0;
    for (const num1 of nums) {
        for (const num2 of nums) {
            if (num1 > num2) {
                smallerNumbersCount++;
            }
        }
        answer.push(smallerNumbersCount);
        smallerNumbersCount = 0;
    }
    return answer;
};

